src/main/java/com/microsoft/azure/AzureVMCloudVerificationTask.java

```diff
@@ -67,130 +67,124 @@ public AzureVMCloudVerificationTask() {
     public void execute(final TaskListener arg0) throws IOException, InterruptedException {
         LOGGER.log(Level.INFO, "AzureVMCloudVerificationTask: execute: start");
         
-        if (cloudNames == null || cloudNames.isEmpty()) {
-            LOGGER.log(Level.INFO, "AzureVMCloudVerificationTask: execute: No clouds, exiting");
-            return;
-        }
-        
-        // Walk the list of clouds and verify the configuration. If an element
-        // is not found (perhaps a removed cloud, removes from the list)
-        synchronized (cloudNamesLock) {
-            List<String> toRemove = new ArrayList<String>();
-            for (String cloudName : cloudNames) {
-                LOGGER.log(Level.INFO, "AzureVMCloudVerificationTask: execute: verifying cloud {0}", cloudName);
-                
-                AzureVMCloud cloud = getCloud(cloudName);
-                
-                // Unknown cloud.  Maybe the name changed since the cloud name
-                // was registered.  Remove from the list
-                if (cloud == null) {
-                    LOGGER.log(Level.INFO, "AzureVMCloudVerificationTask: execute: subscription {0} not found, skipping", cloudName);
-                    // Remove 
-                    toRemove.add(cloudName);
-                    continue;
-                }
-                
-                // If already verified, skip
-                if (cloud.isConfigurationValid()) {
-                    LOGGER.log(Level.INFO, "AzureVMCloudVerificationTask: execute: subscription {0} already verifed", cloudName);
-                    // Update the count.
-                    cloud.setVirtualMachineCount(getVirtualMachineCount(cloud));
-                    continue;
+        if (cloudNames != null && !cloudNames.isEmpty()) {
```

Comment:
It's a bit hard to follow - Did anything change here except for negating the if statement and moving the code inside?